lib/internal/streams/utils.js

```diff
@@ -20,7 +20,7 @@ function isStream(obj) {
 }
 
 function isIterable(obj, isAsync) {
-  if (!obj) return false;
+  if (!obj && typeof obj !== 'string') return false;
```

Comment:
A simpler check would be to check is `obj` is nullish.
```suggestion
  if (obj == null) return false;
```

Reply:
@aduh95 Fixed.